src/robot/utils/misc.py

```diff
@@ -115,3 +115,20 @@ def isatty(stream):
         return stream.isatty()
     except ValueError:    # Occurs if file is closed.
         return False
+
+def parse_re_flags(flags_as_str):
+    result = 0
+    if flags_as_str:
+        unknown = []
+        for f in flags_as_str.split('|'):
```

Comment:
We haven't generally used one character variable names except with comprehensions. Using `for flag in flags.split('|')` or possibly `for name in flags.split('|')` would be better.